HealthVault/Classes/HTTP/MHVNetworkReachability.m

```diff
@@ -137,78 +117,78 @@ -(BOOL)startMonitoring
     MHVCHECK_SUCCESS([self enableCallback:TRUE]);
     MHVCHECK_SUCCESS([self enableNotifications:TRUE]);
     
-    m_isMonitoring = TRUE;
+    self.isMonitoring = TRUE;
     
     return TRUE;
 }
 
--(BOOL)stopMonitoring
+- (BOOL)stopMonitoring
 {
-    if (!m_isMonitoring)
+    if (!self.isMonitoring)
     {
         return TRUE;
     }
     
     MHVCHECK_SUCCESS([self enableNotifications:FALSE]);
     MHVCHECK_SUCCESS([self enableCallback:FALSE]);
     
-    m_isMonitoring = FALSE;
+    self.isMonitoring = FALSE;
     
     return TRUE;
 }
 
--(void)broadcastStatusChange:(SCNetworkConnectionFlags)flags
+- (void)broadcastStatusChange:(SCNetworkConnectionFlags)flags
 {
     BOOL shouldNotify = FALSE;
+    
     @synchronized(self)
     {
-        shouldNotify = (flags != 0 && m_status != flags);
-        m_status = flags;
+        shouldNotify = (flags != 0 && self.status != flags);
+        self.status = flags;
     }
     if (shouldNotify)
     {
-        [[NSNotificationCenter defaultCenter]
-         postNotificationName: MHVHostReachabilityNotificationName
-         object:self
-         ];
+        [[NSNotificationCenter defaultCenter] postNotificationName:MHVHostReachabilityNotificationName
```

Comment:
Let's get rid of the notifications before someone starts using them. We should just inject the network observer into the classes that need it. Notifications cause problems with tests, and can open the door for less than ideal architecture.